npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker/version_resolver.rb

```diff
@@ -650,7 +658,7 @@ def version_install_arg(version:)
           git_source = dependency.requirements.find { |req| req[:source] && req[:source][:type] == "git" }
 
           if git_source
-            "#{dependency.name}@#{git_req[:source][:url]}##{version}"
+            "#{dependency.name}@#{git_source[:source][:url]}##{version}"
```

Comment:
I think this may have been a typo, but I'd like someone to double check my change.

Reply:
@JamieMagee Agreed that it was likely a typo. I've grepped around and not found any definition of `git_req` in the NPM ecosystem code 🙂